src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -2072,6 +2036,12 @@ void Llvm::buildReturn(GenTree* node)
 {
     assert(node->OperIs(GT_RETURN, GT_RETFILT));
 
+    if (node->OperIs(GT_RETFILT) && _compiler->ehGetBlockHndDsc(CurrentBlock())->HasFaultHandler())
```

Comment:
It would be kind of nice to separate `RETFILT`-the-filter-return from `RETFILT`-the-fault-return...